build.sbt

```diff
@@ -616,18 +667,17 @@ lazy val test = project
     fork in IntegrationTest := true,
     javaOptions in IntegrationTest += "-Xmx1G",
     testFrameworks += new TestFramework("scala.tools.partest.sbt.Framework"),
-    testOptions in IntegrationTest += Tests.Setup( () => root.base.getAbsolutePath + "/pull-binary-libs.sh" ! ),
```

Comment:
This is expected. You still need to run `pull-binary-libs.sh` initially. Note that even with the now removed line it used to fail on the first run because doing it in `Test.Setup` is too late (the classpath has already been built before the JARs are downloaded). Only subsequent runs would succeed.

But I don't really want to call it from the sbt build at all because that would require a Unix environment. Since most of the binaries are already resolved through Ivy, I hope that we can get of it entirely. JSoup is on Maven Central, the remaining binary files are about 10 kB in total. It would be nice to generate them on demand but the size is so low that we could just check them into git. Another option is to download them with Scala code in the sbt build.